MARKET_ADD.md

```diff
@@ -82,33 +82,37 @@ To get local tests running a chainfork with the new proposal type, set the `PROP
 
 Once this has been set, then integration tests can be created which ensure the proposal is working as expected.
 
-If deploying and generating calldata for the first time, environment variable `DO_AFTER_DEPLOY_MTOKEN_BROADCAST` should be set to true. After doing deploy and setting the addresses in `Addresses.sol`, this variable should be set to false. This variable is used to determine whether or not to broadcast the after deploy transactions that configure the MToken, which are not needed after the tokens are deployed.
+If deploying and generating calldata for the first time, environment variable `DO_AFTER_DEPLOY_MTOKEN_BROADCAST` should be set to true. After doing deploy and setting the addresses in `Addresses.json`, this variable should be set to false. This variable is used to determine whether or not to broadcast the after deploy transactions that configure the MToken, which are not needed after the tokens are deployed.
```

Comment:
That seems like a good improvement. But can we move away from environment variables? Or if needed, the script should set those variables so that they are codified.